rcl/test/rcl/test_validate_enclave_name.cpp

```diff
@@ -47,6 +53,57 @@ TEST(TestValidateEnclaveName, test_validate) {
     RCL_RET_OK,
     rcl_validate_enclave_name("/foo/bar", &validation_result, &invalid_index));
   EXPECT_EQ(RCL_ENCLAVE_NAME_VALID, validation_result);
+
+  {
+    auto mock = mocking_utils::patch(
+      "lib:rcl", rmw_validate_namespace_with_size,
+      [&](auto, auto, int * result, size_t * index) {
+        if (index) {
+          *index = 0u;
+        }
+        *result = RMW_NAMESPACE_INVALID_TOO_LONG;
+        return RMW_RET_OK;
+      });
+
+    EXPECT_EQ(
+      RCL_RET_OK,
+      rcl_validate_enclave_name("/foo/baz", &validation_result, &invalid_index));
+    EXPECT_EQ(RCL_ENCLAVE_NAME_VALID, validation_result);
+  }
+}
+
+TEST(TestValidateEnclaveName, test_validate_on_internal_error) {
+  int validation_result;
+  size_t invalid_index;
```

Comment:
Should these be initialized?

Reply:
We could, but these are dummy variables. We have no use for their values.